### ISSUE TITLE
Option to skip intro without popup

### ISSUE THREAD

**Issue by tydog98** [jellyfin/jellyfin-androidtv#4277]
### Describe the feature you'd like

Sometimes I'd like to skip the intro, but other times I'd like to watch it without the popup being in the way.

**Comment by nielsvanvelzen** [comment]
You can change the intro segment type to always skip instead of ask to skip.

**Comment by tydog98** [comment]
Yes, but I am not looking to skip all the time, only sometimes.

**Comment by nielsvanvelzen** [comment]
Then use the default ask to skip option?

**Comment by tydog98** [comment]
Yes, but the default has the overlay that blocks part of the image. It would be nice to have the default option but without the overlay.

**Comment by nielsvanvelzen** [comment]
You have 3 options to choose from right now, skipping, asking to skip and doing nothing. You can't ask to skip without asking, you can't skip with asking.

**Comment by MetalDroid** [comment]
I would like a fourth option, Auto skipping with option to cancel from popup: "skipping in 5, 4, 3..., tap to cancel".

It could be interesting, although obviously not a priority implementation, but just an idea.

I haven't tried auto skipping, as I'm very happy with ask to skip.
